src/qtmozembed/declarativewebpage.cpp

```diff
@@ -95,6 +95,12 @@ DeclarativeWebPage::DeclarativeWebPage(QObject *parent)
     connect(&m_grabWritter, SIGNAL(finished()), this, SLOT(grabWritten()));
     connect(this, SIGNAL(urlChanged()), this, SLOT(onUrlChanged()));
     connect(this, &QOpenGLWebPage::contentHeightChanged, this, &DeclarativeWebPage::updateViewMargins);
+    connect(this, &QOpenGLWebPage::loadedChanged, this, [=]() {
```

Comment:
I guess this form of `connect` should accept only 3 arguments: the second `this` is not needed. Also it's recommended to avoid capturing the whole context: `this` instead of `=` should be enough.